### PR TITLE
Add new candidate for Micronaut

### PR DESCRIPTION
**Pull request by jameskleeh**
We won't have the 1.0.0.M1 release for another month or so, however I just wanted to get this in place.

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@ChangeLog(order = "009")
+class MicronautMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_micronaut_1_0_0_M1", author = "jameskleeh")
+  def migration001(implicit db: MongoDatabase) = {
+    insertCandidate(
+      Candidate(
+        candidate = "micronaut",
+        name = "Micronaut",
+        description = "Micronaut is an open source microservice framework for the JVM",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala]: Can we be a bit more elaborate with the description? This is what will appear on the [sdks page](http://beta.sdkman.io/sdks) of our site, as well ash when you do an `sdk ls` on the command line.

File: src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        default = "1.0.0.M1",
+        websiteUrl = "http://micronaut.io/",
+        distribution = "UNIVERSAL"))
+  }
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala]: We will also need an `insertVersion()` to fulfill the initial candidate creation.

File: src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala (hunk 1 above)
Comment (jameskleeh) [reply src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala]: @marc0der Thank you for letting me know. I wasn't aware of that. I'll update it

**Comment by marc0der** [comment]
I've done some changes in the SDKMAN backend that will now allow for a candidate to exist _without_ setting a default version. I've subsequently updated your PR and it has now been merged so closing this PR (somehow github seems confused about this).

**Pull request closed without merging**